.travis.yml

```diff
@@ -37,7 +37,7 @@ before_script:
 # Install composer dependencies
   - composer validate
   - composer install --prefer-dist
-  - composer require --prefer-dist --no-update silverstripe/recipe-core:4.x-dev silverstripe/versioned:1.x-dev
+  - composer require --prefer-dist --no-update silverstripe/recipe-cms:4.4.x-dev
```

Comment:
Do we really need the whole recipe here? Seems like it could just be `versioned` and `cms`, right? I'd rather not slow down the builds and keep the tests as isolated as possible.

Reply:
We just need CMS I think because we're using SiteTree objects for testing.

Reply:
Right, that's what I'm saying. Why use the whole recipe, rather than just `silverstripe/cms`?

Reply:
Build is complaing that we don't have a `Page` class, which we'll only get if using the recipe.

We can untangle this later.